tests/config/nf-test.config

```diff
@@ -31,6 +31,7 @@ profiles {
     docker {
         docker.enabled = true
         docker.userEmulation = false
```

Comment:
```suggestion
```
I'd say we can get rid of this line given it's being deprecated

Reply:
done!